src/locales/en-GB/pages.yaml

```diff
@@ -274,6 +283,19 @@ join:
 
     The chamber music department is always looking for new members. This part of the association is a 'by members for members' department. Here, you can be as active as you'd like and you can perform together with acquaintances at our chamber music concerts. If you'd like to perform, you can organize your performance yourself and gather a group of fun and enthusiastic people to make beautiful and inspiring music.
 
+open-rehearsals:
+  banner:
+    title: Open Rehearsals and Auditions
+    more: More info
+    text: |
+      The orchestra will start with a new programm in March, which means there will be a new round of open rehearsals and auditions. For all info, click the button below.
```

Comment:
*programme?

And I think "click on" (instead of only "click") is better style, but I'm not sure